src/pages/workspace/workflows/WorkspaceWorkflowsPage.tsx

```diff
@@ -51,8 +54,32 @@ function WorkspaceWorkflowsPage({policy, betas, route}: WorkspaceWorkflowsPagePr
 
     const onPressAutoReportingFrequency = useCallback(() => Navigation.navigate(ROUTES.WORKSPACE_WORKFLOWS_AUTOREPORTING_FREQUENCY.getRoute(policy?.id ?? '')), [policy?.id]);
 
-    const items: ToggleSettingOptionRowProps[] = useMemo(
-        () => [
+    const authorizedPayerEmail = policy?.reimburserEmail;
+    const displayNameForAuthorizedPayer = useMemo(
+        () => PersonalDetailsUtils.getPersonalDetailByEmail(authorizedPayerEmail ?? '')?.displayName ?? authorizedPayerEmail,
+        [authorizedPayerEmail],
+    );
+
+    const fetchData = useCallback(() => {
+        if (!policy?.id) {
+            return;
+        }
+        Policy.openWorkspaceReimburseView(policy?.id);
```

Comment:
We'll create a new command `OpenPolicyWorkflowsPage` that will be called instead

Reply:
this API is in produciton